.github/actions/build-sign-publish-chainlink/action.yml

```diff
@@ -51,23 +51,11 @@ inputs:
     description: When set to the string boolean value of "true", the resulting build image will be signed
     default: "false"
     required: false
-  cosign-private-key:
```

Comment:
If we go with this, we should remove the current secret on the repo.

cc @javuto